pip_audit/service/interface.py

```diff
@@ -10,6 +11,11 @@ class Dependency:
     package: str
     version: Version
 
+    # TODO(ww): Use functools.cached_property when supported Python is 3.8+.
+    @property
+    def canonicalized_package(self) -> str:
```

Comment:
```suggestion
    def canonicalized_name(self) -> str:
```
Bit of a nit but I think this should be 'name' and not 'package' (same with the attribute above) -- 'package' is just way too vague.

Reply:
Yep, agreed. I'll change everything over to `name`.